manifests/m/Microsoft/DotNet/AspNetCore/6/6.0.30/Microsoft.DotNet.AspNetCore.6.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.DotNet.AspNetCore.6
+PackageVersion: 6.0.30
+PackageLocale: en-US
+Publisher: Microsoft Corporation
+PackageName: Microsoft ASP.NET Core Runtime 6.0
+PackageUrl: https://dotnet.microsoft.com

```

Comment:
```suggestion
# PackageUrl: 
```
This might be due to a network block of data centers, to prevent automated downloads.